### ISSUE TITLE
race condition in compilation "boost/shared_ptr.hpp"

### ISSUE THREAD

**Issue by arnfinn** [PCMSolver/pcmsolver#24]
```
/home/arnfinn/dalton-openrsp-pcm/external/pcmsolver/src/utils/Cxx11Workarounds.hpp(77): catastrophic error: cannot open source file "boost/shared_ptr.hpp"
  #include <boost/shared_ptr.hpp>
                                 ^

compilation aborted for /home/arnfinn/dalton-openrsp-pcm/external/pcmsolver/src/cavity/Cavity.cpp (code 4)
make[5]: *** [src/cavity/CMakeFiles/cavity.dir/Cavity.cpp.o] Error 4
make[5]: *** Waiting for unfinished jobs....
[ 52%] Building CXX object CMakeFiles/xcfun.dir/src/functionals/optxcorr.cpp.o
/home/arnfinn/dalton-openrsp-pcm/external/pcmsolver/src/utils/Cxx11Workarounds.hpp(77): catastrophic error: cannot open source file "boost/shared_ptr.hpp"
  #include <boost/shared_ptr.hpp>
                                 ^

compilation aborted for /home/arnfinn/dalton-openrsp-pcm/external/pcmsolver/src/interface/Meddle.cpp (code 4)
make[5]: *** [src/interface/CMakeFiles/interface.dir/Meddle.cpp.o] Error 4
make[4]: *** [src/interface/CMakeFiles/interface.dir/all] Error 2
make[4]: *** Waiting for unfinished jobs....
[ 53%] Building CXX object CMakeFiles/xcfun.dir/src/functionals/p86c.cpp.o
/home/arnfinn/dalton-openrsp-pcm/external/pcmsolver/src/utils/Cxx11Workarounds.hpp(77): catastrophic error: cannot open source file "boost/shared_ptr.hpp"
  #include <boost/shared_ptr.hpp>
                                 ^

compilation aborted for /home/arnfinn/dalton-openrsp-pcm/external/pcmsolver/src/cavity/Element.cpp (code 4)
make[5]: *** [src/cavity/CMakeFiles/cavity.dir/Element.cpp.o] Error 4
make[4]: *** [src/cavity/CMakeFiles/cavity.dir/all] Error 2
```

**Comment by robertodr** [comment]
@arnfinn Closed via [v1.1.1](https://github.com/PCMSolver/pcmsolver/tree/v1.1.1) Updating the version in DALTON soon.

**Comment by rahusona** [comment]
Hi @robertodr   arnfinn  - can you please help me how you resolved this issue?
